plugin/src/main/scala/ch/jodersky/sbt/jni/build/BuildTool.scala

```diff
@@ -37,6 +37,7 @@ trait BuildTool {
 
       baseDirectory.mkdir()
       val out = baseDirectory.toPath().resolve(name)
+      Files.createDirectories(out.getParent)
```

Comment:
I think this is necessary, so that the possibly deeply nested structure of directories is created before trying creating a new file.